src/commands/lore/lore.ts

```diff
@@ -66,6 +66,10 @@ export default class Lore implements ICommand {
     async run(ctx: IContext) {
         const name = ctx.getOption("name")?.toLowerCase() || "";
         const output = await this.getOutput(name);
-        output.type === "text" ? ctx.replyText(output.result as string) : ctx.replyEmbed(output.result as Embed);
+        if (output.type === "text"){
+            ctx.replyText(output.result as string)
+            return
+        }
+        ctx.replyEmbed(output.result as Embed);
```

Comment:
Please end lines with `;` to keep consistancy with the rest of the code.

As this method does not return anything, is not a guard clause and won't be indented more than 2 tabs, I also suggest using a `else` statement and removing the `return` keyword